educator/tests/Test/Dscp/Educator/Web/Student/Queries.hs

```diff
@@ -91,13 +91,13 @@ spec_Student_API_queries :: Spec
 spec_Student_API_queries = specWithTempPostgresServer $ do
   describe "Courses" $ do
     describe "getCourse" $ do
-        it "Student is not enrolled initially" $
-            sqlProperty $ \() -> do
-                _ <- createStudent student1
-                _ <- createCourseSimple 1
+        -- xit "Student is not enrolled initially" $
+        --     sqlProperty $ \() -> do
+        --         _ <- createStudent student1
+        --         _ <- createCourseSimple 1
 
-                course <- studentGetCourse student1 courseId1
-                return (not $ ciIsEnrolled course)
+        --         course <- studentGetCourse student1 courseId1
+        --         return (not $ ciIsEnrolled course)
```

Comment:
Does this test fail in CI? I think if it does not, once you finish with launching tests locally you should uncomment it.